doc/reference/tooling/tt_cli/import.rst

```diff
@@ -11,14 +11,19 @@ Importing data
 
 .. code-block:: console
 
-    $ tt [crud] import URI FILE:SPACE [IMPORT_OPTION ...]
+    $ tt [crud|tdg2] import URI FILE:SPACE [IMPORT_OPTION ...]
     # or
-    $ tt [crud] import URI :SPACE < FILE [IMPORT_OPTION ...]
+    $ tt [crud|tdg2] import URI :SPACE < FILE [IMPORT_OPTION ...]
 
-``tt [crud] import`` imports data from a file to a space.
-The ``crud`` command is optional and can be used to import data to a cluster by using the `CRUD <https://github.com/tarantool/crud>`_ module. Without ``crud``, data is imported using the :ref:`box.space <box_space>` API.
+``tt [crud|tdg] import`` imports data from a file to a space. Three import commands
+cover the following cases:
 
-This command takes the following arguments:
+*   ``tt import`` imports data into a single-replicaset storage through its master instance using the :ref:`box.space <box_space>` API.
+*   ``tt crud import`` imports data into a sharded cluster through CRUD router using the `CRUD <https://github.com/tarantool/crud>`_ module.
```

Comment:
> through CRUD router

I'd say `... through a router using the CRUD module` or just `... through a CRUD router`. `CRUD` two times in a row looks excessive.